sql/views.sql

```diff
@@ -76,7 +76,27 @@ SELECT ht.schema_name AS hypertable_schema,
   END AS next_start,
   js.total_runs,
   js.total_successes,
-  js.total_failures
+  js.total_failures,
+  CASE WHEN js.total_successes = 0 THEN
+	NULL
+  ELSE 
+  	js.total_successful_duration
+  END AS total_successful_duration,
```

Comment:
```suggestion
  NULLIF(js.total_successes, 0) AS total_successful_duration,
```

For better readability

Reply:
This won't work, it would pass through `js.total_successes`  when `=0`, but I'm passing through `js.total_successful_duration`

Reply:
I mean I could do `NULLIF(js.total_successful_duration, js.total_successful_duration + js.total_successes)` - but that feels a bit wonky

Reply:
I think it's probably fine just to start them at zero and not worry about NULLing?

Reply:
Ahh you're correct